### PR TITLE
Replace bobFacts with menuSuggestion in rag docs

### PR DESCRIPTION
**Pull request by tagboola**
I punted on completely revamping the docs and just replaced the instances of bob facts with menu examples

### CODE REVIEW

File: docs/rag.md [hunk 1]
```diff
@@ -124,10 +124,6 @@ Embedding model support is provided through the following plugins:
 The following examples show how you could ingest a collection of PDF documents
 into a vector database and retrieve them for use in a flow.
 
-It uses the local file-based vector similarity retriever
-that Genkit provides out-of-the box for simple testing and prototyping (_do not
-use in production_)
-
```
Comment (tagboola) [inline docs/rag.md:130]: This is duplicated below on line 169.

**Review by MaesterChestnut — APPROVED** [review]
Perfect!

**Review by MichaelDoyle — APPROVED** [review]

File: docs/rag.md [hunk 2]
```diff
@@ -124,10 +124,6 @@ Embedding model support is provided through the following plugins:
 The following examples show how you could ingest a collection of PDF documents
 into a vector database and retrieve them for use in a flow.
```
Comment (MichaelDoyle) [inline docs/rag.md:125]: Consider adding a blurb about what the sample(s) do (read in restaurant menus and answer questions about them).

Comment (tagboola) [reply docs/rag.md]: Done. PTAL.

**Review by kevinthecheung — APPROVED** [review]

**Pull request merged**